test/E2ETests/DiscoveryAndExecutionTests/DiscoveryAndExecutionTests.csproj

```diff
@@ -19,6 +19,10 @@
     <PackageReference Include="System.ValueTuple" Version="$(SystemValueTupleVersion)" />
   </ItemGroup>
 
+  <ItemGroup>
+    <ProjectReference Include="$(RepoRoot)test\TestFramework.ForTestingMSTest\TestFramework.ForTestingMSTest.csproj" />
+  </ItemGroup>
+
   <ItemGroup>
     <Reference Include="Microsoft.VisualStudio.TestPlatform.MSTest.TestAdapter">
```

Comment:
Could you make a test to see if we need to manually reference all these dlls? (Make a run where you try to remove them).

Reply:
if I remove it, I get compilation errors

Reply:
Ok so let's keep things as-is. Thanks for the confirmation.